1-python-question.py

```diff
@@ -25,8 +25,27 @@
 
 
 def processRequest(request):
-    # Your code here
-    return
+    try:
+        if 'applicants' not in request or len(request['applicants'])==0:
+            raise ValueError
+    except:
+        return {'error':'empty'}
+    v=5
+    m=3.5
+    banned=list(filter(lambda b:b if b in bannedVisitors else None,request['applicants']))
+    allowed=list(filter(lambda s:s if s not in bannedVisitors else None,request['applicants']))
```

Comment:
lambda expressions can be simplified to `lambda b : b in bannedVisitors` and `lambda s : s not in bannedVisitors`